source/about-mongosync.txt

```diff
@@ -126,6 +126,20 @@ Finalize Sync
 To finalize the sync between the source and destination clusters, 
 call the :ref:`commit <c2c-api-commit>` endpoint. 
 
+.. note::
+
+   You must block writes to the source cluster before you begin the
+   the commit process. 
+
+   If you set ``enableUserWriteBlocking`` to ``true`` when you used the
```

Comment:
[tense nit]

Consider updating to present tense if possible. Thanks!

Reply:
I left this in past tense because it relates to a user's historical behavior (previously setting `enableUserWriteBlocking` to `true`) - I added the word "previously" to make that more clear

Reply:
That makes sense -- thanks!